skimage/segmentation/chanvese.py

```diff
@@ -0,0 +1,166 @@
+import numpy as np
+from scipy.ndimage import distance_transform_cdt
+
+
+def _cv_curvature(phi):
+    """ Returns the 'curvature' of a level set.
```

Comment:
As far as this and functions below are supportive, it's not neccessary to add detailed dosctring, but you might consider adding references between the objects in docstring and function arguments.
As example, change
```
Returns the 'curvature' of a level set.
```
to
```
Returns the 'curvature' of a level set `phi`.
```

Please, also, consider this change for other supportive functions.